core/src/main/java/com/linecorp/armeria/server/ReconfigurableServer.java

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright 2021 LINE Corporation
+ *
+ *  LINE Corporation licenses this file to you under the Apache License,
+ *  version 2.0 (the "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at:
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ */
+package com.linecorp.armeria.server;
+
+/**
+ * A Server that allows ability to reconfigure its service endpoints without having to restart the server.
+ */
+@FunctionalInterface
+public interface ReconfigurableServer {
+    /**
+     * Reconfigures the server using the specified {@link ServerBuilder}.
+     * @param sb ServerBuilder.
```

Comment:
If I remove the java doc check style complains about it.

Reply:
Hm, that's weird because we don't have to have that.
https://github.com/line/armeria/blob/master/dropwizard2/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaServerConfigurator.java#L29
Do you use another settings?

Reply:
I misunderstood which line you wanted me to delete. I removed the line and checkstyle passes now.